lin.opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+name:         "lin"
+synopsis:     "Experimental library for linearizability checking in Multicore"
+maintainer:   ["Jan Midtgaard <mail@janmidtgaard.dk>"]
+authors:      ["Jan Midtgaard <mail@janmidtgaard.dk>"]
+license:      "BSD-2-clause"
+homepage:     "https://github.com/jmid/multicoretests"
+bug-reports:  "https://github.com/jmid/multicoretests/issues"
+dev-repo:     "git+https://github.com/jmid/multicoretests.git"
+depends: [
+  "base-domains"
+  "dune"                {>= "2.9.3"}
+  "ocamlfind"           {>= "1.9.3"}
+  "domainslib"          {>= "0.4.0"}
+  "qcheck-core"         {>= "0.18.1"}
+  "qcheck"              {>= "0.18.1"}
+  "odoc"                {with-doc}
+]
+pin-depends: [
+  ["domainslib.0.4.0"          "git+https://github.com/ocaml-multicore/domainslib.git#master"]
+  ["sexplib0.v0.14.0"          "git+https://github.com/patricoferris/sexplib0#5.00"]
+  ["ppxlib.0.25.0~5.00preview" "git+https://github.com/kit-ty-kate/ppxlib#500+sexp"]
```

Comment:
I think `sexplib0` and `ppxlib` can be removed here.

Pinning recent versions of these two is needed for
- `ppx_deriving` - to auto-derive `show`
- `ppx_deriving_qcheck` - to auto-derive generators
if I remember correctly.

A user should be able to use the library with hand-written versions of the above, but both are just very convenient.
I guess that information would be good to include in the README...

Reply:
Adressed.